wagl/ancillary.py

```diff
@@ -217,10 +218,25 @@ class NbarPathsDict(TypedDict):
     brdf_dict: BrdfDict
 
 
+def is_offshore_territory(acq, offshore_territory_boundary_path):
+    geobox = acq.gridded_geo_box()
+    acq_polygon = Polygon(
+        [geobox.ul_lonlat, geobox.ur_lonlat, geobox.lr_lonlat, geobox.ll_lonlat]
+    )
+
+    with fiona.open(offshore_territory_boundary_path, "r") as offshore_territory:
+        for boundary_poly in offshore_territory:
+            if shape(boundary_poly["geometry"]).contains(acq_polygon):
```

Comment:
To confirm: we do want _contains_, not _intersects_?

The acquisitions never fully line up, there's jitter, and if I'm reading correctly, an acq at that 98% covers the existing territory will be considered an off-short territory here?

(btw, I don't know how far the nearest off-shore territory is — whether boundaries issues like this actually matter)